src/utils/daos_hdlr.h

```diff
@@ -69,11 +92,22 @@ struct cmd_args_s {
 	enum pool_op		p_op;		/* pool sub-command */
 	enum cont_op		c_op;		/* cont sub-command */
 	enum obj_op		o_op;		/* obj sub-command */
+	enum fs_op		fs_op;		/* filesystem sub-command */
 	char			*sysname;	/* --sys-name or --sys */
 	uuid_t			p_uuid;		/* --pool */
+	uuid_t			src_p_uuid;	/* --src_pool */
+	uuid_t			dst_p_uuid;	/* --dst_pool */
+	uuid_t			src_cont_uuid;	/* --src_cont */
+	uuid_t			dst_cont_uuid;	/* --dst_cont */
```

Comment:
```suggestion
	uuid_t			src_c_uuid;	/* --src_cont */
	uuid_t			dst_c_uuid;	/* --dst_cont */
```

While I do prefer `cont`, the existing `--cont` using `c`.
Maybe we should be consistent?